plugins/webapi/mana/online.go

```diff
@@ -0,0 +1,44 @@
+package mana
+
+import (
+	"net/http"
+
+	manaPkg "github.com/iotaledger/goshimmer/packages/mana"
+	manaPlugin "github.com/iotaledger/goshimmer/plugins/mana"
+	"github.com/labstack/echo"
+)
+
+func getOnlineAccessHandler(c echo.Context) error {
+	return getOnlineHandler(c, manaPkg.AccessMana)
+}
+
+func getOnlineConsensusHandler(c echo.Context) error {
+	return getOnlineHandler(c, manaPkg.ConsensusMana)
+}
+
+// getOnlineHandler handles the request.
+func getOnlineHandler(c echo.Context, manaType manaPkg.Type) error {
+	onlinePeersMana, err := manaPlugin.GetOnlineNodes(manaType)
+	if err != nil {
+		return c.JSON(http.StatusNotFound, GetOnlineResponse{Error: err.Error()})
```

Comment:
Don't shoot me LOL. :)

The error that is returned here is from `bmv.GetMana` which returns error on 2 cases. 
If the nodeID is not found in the map or if an invalid weight param is used.

Therefore, I don't think returning `http.StatusNotFound` for either of these cases is appropriate.
A more general `http.StatusBadRequest` could be used.

Reply:
My thinking was that there is nothing wrong with the request, it is the requested resource that has problems in both cases (not in map or invalid weight parameter). It's not the client's fault, rather something in the server, hence this status code.

Btw because of how `GetOnlineNodes` is implemented, we won't see `node not found` errors (mana value will just be `0`), and the only way to get an `invalid weight parameter` is if that hard coded `0.5` is changed in the source code for something invalid.

Reply:
I see your point.